pages/common/meshnamed.md

```diff
@@ -0,0 +1,16 @@
+# meshname
```

Comment:
```suggestion
# meshnamed
```
Maybe, we could update the Heading to match with the filename and examples?

Reply:
Thanks for the hint!